packages/typespec-test/test/widget_dpg/generated/typespec-ts/package.json

```diff
@@ -46,7 +46,7 @@
   ],
   "dependencies": {
     "@azure/core-util": "^1.9.2",
-    "@azure-rest/core-client": "^2.1.0",
+    "@azure-rest/core-client": "^2.3.0",
```

Comment:
@timovv our test cases failed with following error. And I created an issue here, https://github.com/Azure/azure-sdk-for-js/issues/31349. Could you help fix this?

![image](https://github.com/user-attachments/assets/1f19885c-ed66-4ef3-a6b5-d44ce0e6be9b)

![image](https://github.com/user-attachments/assets/3dc24404-c733-4852-93c7-705d5578fde4)